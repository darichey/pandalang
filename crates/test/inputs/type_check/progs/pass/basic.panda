let x = 3

let y = "hello"

let z = x + 5
