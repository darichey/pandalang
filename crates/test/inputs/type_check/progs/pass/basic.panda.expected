Ok(
    [
        (
            "x",
            "Int",
        ),
        (
            "y",
            "Str",
        ),
        (
            "z",
            "Int",
        ),
    ],
)